pytorch_blade/pytorch_blade/compiler/jit/shape_type_spec.cpp

```diff
@@ -97,11 +97,11 @@ ShapeType ShapeType::Deserialize(const std::string& serial_str) {
         // string like: 1:1 (min:max)
         CHECK(!s.empty());
         std::vector<std::string> dim_range = split(s, ":");
-        CHECK_EQ(dim_range.size(), 2);
+        CHECK(dim_range.size() == 2);
```

Comment:
`CHECK` and `TORCH_CHECK` are both used in this file ...  so where is `CHECK` from ... and why `CHECK` is used...